x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/FieldAttributeTests.java

```diff
@@ -431,4 +464,22 @@ public void testExpandStarOnIndexWithoutColumns() {
         assertTrue(((Project) plan).projections().isEmpty());
     }
 
+    private static IndexResolution loadMapping(String mappingName) {
```

Comment:
It helps to rename this method since it clashes with TypesTest.loadMapping and it's confusing to see the first returns a mapping while the latter an IndexResult.
How about `loadIndexResolution`?

Reply:
Sure, will do (along with the needed `INTRODUCING_UNSIGNED_LONG` update after the pending version bump).